README.md

```diff
@@ -11,9 +11,9 @@
 
 See the AUTHORS file for the list of authors.
 
-And many more people helped on the SymPy mailing list, reported bugs,
-helped organize SymPy's participation in the Google Summer of Code, the
-Google Highly Open Participation Contest, Google Code-In, wrote and
+And many more people who helped on the SymPy mailing list, in reporting bugs,
+helped to organize SymPy's participation in the Google Summer of Code(GSoC), the
+Google Highly Open Participation Contest, Google Code-In, have written and
```

Comment:
This grammar isn't really an improvement. The `who` makes sense, but the tense is now weird.

I think this should just be
```suggestion
Many more people helped on the SymPy mailing list, reported bugs,
helped organize SymPy's participation in the Google Summer of Code, the
Google Highly Open Participation Contest, Google Code-In, wrote and
```
Where the only change I made from the original was to remove the word "and"